drivers/spi/Kconfig.stm32

```diff
@@ -0,0 +1,30 @@
+# Kconfig - STM32 SPI driver configuration options
+
+#
+# Copyright (c) 2015-2016 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig SPI_STM32
+	bool "STM32 MCU SPI controller driver"
+	depends on SPI && SOC_FAMILY_STM32
+	depends on SOC_SERIES_STM32L4X || SOC_SERIES_STM32F4X || SOC_SERIES_STM32F3X
```

Comment:
Why did you leave out the previous dependencies and selections?
```
depends on SPI && SOC_FAMILY_STM32
depends on SOC_SERIES_STM32L4X || SOC_SERIES_STM32F0X || SOC_SERIES_STM32F3X || SOC_SERIES_STM32F4X
select HAS_DTS_SPI
select USE_STM32_LL_SPI
```

Reply:
looks like one of the many rebase I did that missed this.

Reply:
Actuall, that explains the DTS issue as well, since select HAS_DTS_SPI was missing ...